src/util/format_expr.cpp

```diff
@@ -128,12 +129,18 @@ static std::ostream &format_rec(
     else
       return os << src.pretty();
   }
-  else if(type == ID_unsignedbv || type == ID_signedbv)
+  else if(type == ID_unsignedbv ||
+          type == ID_signedbv ||
+          type == ID_c_bool)
     return os << *numeric_cast<mp_integer>(src);
   else if(type == ID_integer)
     return os << src.get_value();
+  else if(type == ID_string)
+    return os << '"' << escape(id2string(src.get_value())) << '"';
```

Comment:
Wishlist: I think the `escape` function should really be moved to `string_utils.{h,cpp}`.

Reply:
done